SECURITY.md

```diff
@@ -0,0 +1,126 @@
+# Security Policy
+
+## Introduction
+
+Codenotary, September 7th 2022
+
+*This security policy is based on [this template](https://www.cisa.gov/vulnerability-disclosure-policy-template).*
+
+Codenotary is committed to ensuring the security of the American and global public by protecting their information. This policy is intended to give security researchers clear guidelines for conducting vulnerability discovery activities and to convey our preferences in how to submit discovered vulnerabilities to us.
+
+This policy is addressed to people using the immudb and / or contributing to the project. It describes **what systems and types of research** are covered under this policy, **how to send** us vulnerability reports, and **how long** we ask security researchers to wait before publicly disclosing vulnerabilities.

```

Comment:
> and **how long** we ask security researchers to wait before publicly disclosing vulnerabilities.

We don't specify anywhere in the policy how long we ask security researchers before disclosing the vulnerability.

Reply:
What length of this period would you recommend?

Reply:
I would go for 90 business days, it's usually within 60 to 120 days range.

Reply:
I added a clause about that.